packages/networks/src/networks/sepolia.ts

```diff
@@ -0,0 +1,68 @@
+import { NetworkConfig } from '@unlock-protocol/types'
+
+export const sepolia: NetworkConfig = {
+  blockScan: {
+    url: (address: string) => `https://blockscan.com/address/${address}`,
+  },
+  chain: 'base',
+  description:
+    'Sepolia is the primary testnet recommended by Ethereum for dapp development.',
+  explorer: {
+    name: 'Sepolia Etherscan',
+    urls: {
+      address: (address: string) =>
+        `https://sepolia.etherscan.io/address/${address}`,
+      base: `https://sepolia.etherscan.io/`,
+      token: (address: string, holder: string) =>
+        `https://sepolia.etherscan.io/token/${address}?a=${holder}`,
+      transaction: (hash: string) => `https://sepolia.etherscan.io/tx/${hash}`,
+    },
+  },
+  featured: true,
```

Comment:
```suggestion
  featured: false,
```

Let's set to `false` first so we can test everything works as expected